db/model/room.js

```diff
@@ -51,6 +51,14 @@ module.exports = function room(seq, dataTypes) {
       allowNull: true,
       comment: 'Key/Value pairs for user specific settings',
     },
+    origin: {
+      type: dataTypes.STRING,
+      defaultValue: 'other',
+      validate: {
+        is: constants.nameRegex,
+      },
+      comment: 'Tracks where refocus room was created from',
```

Comment:
why is this comment different from swagger description of same field? I like this one

Reply:
Updated swagger to be consistent with db/model/room.js